services/vision/collector.go

```diff
@@ -0,0 +1,71 @@
+package vision
+
+import (
+	"context"
+
+	servicepb "go.viam.com/api/service/vision/v1"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	"go.viam.com/rdk/data"
+	"go.viam.com/rdk/vision/viscapture"
+)
+
+type method int64
+
+const (
+	CaptureAllFromCamera method = iota
+)
+
+func (m method) String() string {
+	switch m {
+	case CaptureAllFromCamera:
+		return "CaptureAllFromCamera"
+	}
+	return "Unknown"
+}
+
+func NewCaptureAllFromCameraCollector(resource interface{}, params data.CollectorParams) (data.Collector, error) {
+	vision, err := assertVision(resource)
+	if err != nil {
+		return nil, err
+	}
+
+	cFunc := data.CaptureFunc(func(ctx context.Context, _ map[string]*anypb.Any) (interface{}, error) {
+		visCaptureOptions := viscapture.CaptureOptions{
+			ReturnImage:           true,
+			ReturnDetections:      true,
+			ReturnClassifications: true,
+			ReturnObject:          true,
+		}
+		visCapture, err := vision.CaptureAllFromCamera(ctx, "camera-1", visCaptureOptions, nil)
```

Comment:
@Rob1in Is there anyway to get the camera name here?

Reply:
Since it can be any camera, I think it should be configurable in the collector. 
And maybe same for the viscapture.CaptureOptions